### PR TITLE
NetworkFirewall - Fixed LoggingConfiguration

### PR DESCRIPTION
**Pull request by ntwaddell**
The logging configuration was not correct per CloudFormation documentation

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-networkfirewall-loggingconfiguration.html

### CODE REVIEW

**Comment by ntwaddell** [comment]
@markpeek

File: troposphere/networkfirewall.py [hunk 1]
```diff
@@ -114,17 +114,28 @@ class FirewallPolicy(AWSObject):
     }
 
 
-class LogDestinationConfig(AWSProperty):
+class LogDestinationConfigs(AWSProperty):
```
Comment (markpeek) [inline troposphere/networkfirewall.py:117]: According to the docs, this should remain singular.

File: troposphere/networkfirewall.py [hunk 2]
```diff
@@ -114,17 +114,28 @@ class FirewallPolicy(AWSObject):
...
         "LogDestinationType": (str, True),
         "LogType": (str, True),
     }
 
 
-class LoggingConfiguration(AWSProperty):
+class LoggingConfigurationProperty(AWSProperty):
+
+    props = {
+        "LogDestinationConfigs": ([LogDestinationConfigs], True)
```
Comment (markpeek) [inline troposphere/networkfirewall.py:128]: This should be use a singular inside of the list.
```
"LogDestinationConfigs": ([LogDestinationConfig], True)
```

**Review by markpeek — CHANGES_REQUESTED** [review]
Thank you for correcting this Object/Property issue. Looking at the docs `LogDestinationConfig` should remain singular.

File: troposphere/networkfirewall.py (hunk 1 above)
Comment (ntwaddell) [reply troposphere/networkfirewall.py]: Ahh, you are correct.  Updated!

**Review by markpeek — APPROVED** [review]
LGTM

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**